plugins/datastreamer_plugin/ui/topic_selection_dialog/Configuration.hpp

```diff
@@ -42,6 +42,14 @@ namespace ui {
 struct Configuration
 {
 
+    Configuration();
```

Comment:
NIT: Referring to `struct Configuration` doxygen (even though not belongs to this PR):

`every Fast DDS entity required` -> `every required Fast DDS entity`

Minor grammar typo and not clear what `the process requires to instantiate` means:
`It create, manage and destroy every Fast DDS entity that the process requires to instantiated` -> `It creates, manages and destroys every required Fast DDS entity.`?

Reply:
this was a copy/pasted doxygen, it was incorrect.